spec/models/response_spec.rb

```diff
@@ -131,6 +131,20 @@
     end
   end
 
+  describe '#populate_new_response' do
+    context 'when current round response is found' do
+      it 'returns the current round response' do
+        allow(response).to receive(:populate_new_response).with(:review_response_map, "0").and_return(response)
+        expect(response.id).to eq(1)
+      end
+    end
+    context 'when current round response is found' do
+      it 'returns a new response object' do
```

Comment:
This test does not have expectations, therefore, this test will never fail. Please add the expectation for this test.